packages/esm-patient-test-results-app/src/trendline/trendline.component.tsx

```diff
@@ -77,7 +78,10 @@ const Trendline: React.FC<{
 
   const [upperRange, lowerRange] = React.useMemo(() => {
     const dates = patientData[1].map((entry) => new Date(Date.parse(entry.effectiveDateTime)));
-    return [dates[0], dates[dates.length - 1]];
+    const newUpperRangeVal = dates.filter(
+      (result) => dayjs(result).format('MM/DD/YYYY') == dayjs().format('MM/DD/YYYY'),
+    );
+    return [newUpperRangeVal.length > 0 ? dates[0] : new Date(dayjs().format('MM/DD/YYYY')), dates[dates.length - 1]];
```

Comment:
@walteronoh Why is this different from

```suggestion
    return [new Date(dayjs().format('MM/DD/YYYY')), dates[dates.length - 1]];
```

?